Deployment.md

```diff
@@ -64,7 +64,7 @@ After that pick and domain and done 🎉
 ## Gitlab Pages
 
 To deploy to Gitlab Pages add a new job with the title _pages_ in your `.gitlab-ci.yml` configuration file.
-Gitlab Pages hosts your static files at `https://<group-or-user-name>.gitlab.io/<project-name>`.
```

Comment:
You need this otherwise it won't render the okay :/

Reply:
This change got lost in merge. I readded the backpacks